ci/integration_tests/src/tests/cw20_stake_test.rs

```diff
@@ -0,0 +1,118 @@
+use core::time;
+use std::thread;
+
+use crate::helpers::{chain::Chain, helper::create_dao};
+use cosmwasm_std::{to_binary, Uint128};
+use cw20_stake::{msg::StakedValueResponse, state::Config};
+use cw_core_interface::voting::VotingPowerAtHeightResponse;
+use test_context::test_context;
+
+// #### ExecuteMsg #####
+
+#[test_context(Chain)]
+#[test]
+fn execute_stake_tokens(chain: &mut Chain) {
+    let voting_contract = "cw20_staked_balance_voting";
+
+    let res = create_dao(chain, None, "exc_stake_create_dao", chain.user.addr.clone());
+    let dao = res.unwrap();
+
+    let voting_addr = dao.state.voting_module.as_str();
+
+    // stake dao tokens:
+    chain
```

Comment:
maybe we can do this in the create_dao method so other tests can use it later?

Reply:
Yeah, great point. I want to make `create_dao()` more configurable first, and Im going to do that in another PR with more integration tests.